src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -90,15 +90,18 @@ public SSHConfig() throws IOException, DuplicateKeyException, YamlReaderExceptio
                 "The file must be in the OpenSSH format.",
                 "Create keys with 'ssh-keygen -t rsa -b 4096' and add them to the file.",
                 "The genereated file can be found in the .ssh directory of the user that created the key, unless a different path was specified.",
```

Comment:
Can you also add default paths for windows and Linux here.

Reply:
debated doing this, but by default those paths are not present in neither Windows nor Linux kernels. ill make a note of it and add it tho